components/alert.css

```diff
@@ -0,0 +1,38 @@
+.alerts .alert-info,
+.alerts .alert-warn,
+.alerts .alert-error,
+.alerts .alert-success {
+  margin: 1rem 0;
+  padding: 0.5rem 1rem;
+  width: 70%;
+  box-sizing: border-box;
+  border-radius: 3px;
+  font-size: 1.2rem;
+}
```

Comment:
you can make new class for giving this styling to each alert component instead of this nesting CSS selectors.

like this 
```html
<div class="inline alert-info alert">
```

```css
.alert{
  margin: 1rem 0;
  padding: 0.5rem 1rem;
  width: 70%;
  box-sizing: border-box;
  border-radius: 3px;
  font-size: 1.2rem;
}
```

Reply:
right 👍

Reply:
Done, thank you for the suggestion. @viru2001